src/Advertising.js

```diff
@@ -62,15 +81,34 @@ export default class Advertising {
             adUnitCodes: divIds,
             bidsBackHandler: () => {
               window.pbjs.setTargetingForGPTAsync(divIds);
-              Advertising.queueForGPT(
-                () => window.googletag.pubads().refresh(selectedSlots),
-                this.onError
-              );
+              this.requestManager.prebid = true;
+              this.refreshSlots(selectedSlots);
```

Comment:
Why aren't we doing `Advertising.queueForGPT` here anymore? I can't remember what the problem was.

Reply:
the refreshSlots method does queueForGPT, so this should be OK